### PR TITLE
Migrate l10n_ec_ote Ecuador Geopolitical Info

### PR DESCRIPTION
**Pull request by patovala**

### CODE REVIEW

File: l10n_ec_ote/models/ote.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+from odoo import fields, models
+
+
+class Canton(models.Model):
```
Comment (damendieta) [inline l10n_ec_ote/models/ote.py]: This model should be replaced by res.city and depend on the module base_parish that is proposed on the partner-contact repository.

File: l10n_ec_ote/README.rst [hunk 2]
```diff
@@ -0,0 +1,107 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (damendieta) [inline l10n_ec_ote/README.rst:1]: Maybe it's better to use the new methodology and let the OCA bot create the readme based on the files. For example: https://github.com/OCA/project/tree/14.0/project_category/readme

File: l10n_ec_ote/models/ote.py [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+    state_id = fields.Many2one(
+        "res.country.state",
+        ondelete="restrict",
+    )
+    name = fields.Char()
+    code = fields.Char()
+
+
+class Parish(models.Model):
```
Comment (damendieta) [inline l10n_ec_ote/models/ote.py:16]: This model should be replaced by the model on base_parish.

File: l10n_ec_ote/security/ir.model.access.csv [hunk 4]
```diff
@@ -0,0 +1,3 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
```
Comment (damendieta) [inline l10n_ec_ote/security/ir.model.access.csv:1]: This rules are also not required, as all models are declared on the dependencies.

**Review by damendieta — CHANGES_REQUESTED** [review]

**Comment by patovala** [comment]
Closing this PR cause the feature will be merged into other specific module

**Pull request closed without merging**